app/src/main/java/cm/aptoide/pt/home/HomePresenter.java

```diff
@@ -717,4 +723,27 @@ private void handleLoadMoreError() {
           }
         });
   }
+
+  @VisibleForTesting public void handleAppComingSoonClick() {
+    view.getLifecycleEvent()
+        .filter(lifecycleEvent -> lifecycleEvent.equals(View.LifecycleEvent.CREATE))
+        .flatMap(created -> view.notifyMeClicked())
+        .filter(homeEvent -> homeEvent.getBundle() instanceof ActionBundle)
+        .doOnNext(event -> {
+          ActionBundle bundle = (ActionBundle) event.getBundle();
+          homeAnalytics.sendPromotionalArticleClickEvent(bundle.getType()
+              .name(), bundle.getActionItem()
+              .getCardId());
+          homeAnalytics.sendActionItemEditorialTapOnCardInteractEvent(bundle.getTag(),
```

Comment:
As we talked we can probably remove the Editorial from this method name since it is no longer strict to editorial.